### DISCUSSION TITLE
Nordigen: Duplicates have different external_id's. firefly bug or should I take this up with nordigen?

### DISCUSSION THREAD

**Discussion by dekiesel** [firefly-iii/firefly-iii#6573] [Firefly III]
Hi,

I have (some) duplicates that have different external_ids (as seen in the debug and transaction view). Is this something I need to take up with nordigen or would you like to see the two debug json's anyway?

Thanks!

**Comment by JC5** [comment]
I've seen it as well with my transactions. It's Nordigen. I'm not sure of the cause though.

**Reply by dekiesel** [reply]
Thanks for letting me know. Did you send them an email? If not I'll give it a try.

**Reply by JC5** [reply]
Not yet, haven't gotten around to it. If possible, send the debug lines from the importer as well. If you aren't in a position to do so, you'll have to wait for me to get another duplicate.

**Reply by dekiesel** [reply]
I sent them a mail. I'll let you know when I get a response.

**Reply by JC5** [reply]
Thanks!

**Reply by dekiesel** [reply]
`Could you please also share the account_id, where the transaction duplicates occurred? `

The reply from Nordigen. Any idea where I can find that id? Or is it just my IBAN?

**Reply by JC5** [reply]
The Data Importer logs will show you (right at the beginning) from which account it's downloading transactions. The URL is the identifier.

**Reply by dekiesel** [reply]
Just so I understand: That's my bank account number, right? I only have once account connected via Nordigen to firefly.

**Reply by JC5** [reply]
Sorry no it's a different number. Ill look up an example later, I'm on my phone right now.

**Reply by dekiesel** [reply]
`ListAccountsResponse includes account "<SomeUUID>"`

Is it this one?

**Reply by JC5** [reply]
Yes 👍

**Reply by dekiesel** [reply]
Thanks, I updated the ticket with Nordigen.

**Reply by dekiesel** [reply]
The issue has been resolved according to Nordigen, but they'd like me to test ist. Is there an easy way to completely  delete transactions (let's say everything since the october 1th or do I have to fiddle with the database?

**Reply by JC5** [reply]
there is no "purge" button yet unfortunately. you have to fiddle with the database. Table `transaction_journals` is what you need. Let me know if i can help.

**Reply by dekiesel** [reply]
I can confirm that the last nordigen update fixed the issue :) Thank you very much for your help!

While I have you, is there a database diagram somewhere? I'd like to play around with [Metabase](https://www.metabase.com/).

**Reply by JC5** [reply]
There is [this](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/architecture/), but you'll have to use something like MySQL workbench to extract it from a live database.

**Reply by dekiesel** [reply]
That'll do nicely, thank you!
